src/SpiAnalyzer.cpp

```diff
@@ -173,6 +180,13 @@ bool SpiAnalyzer::WouldAdvancingTheClockToggleEnable()
     U64 next_edge = mClock->GetSampleOfNextEdge();
     bool enable_will_toggle = mEnable->WouldAdvancingToAbsPositionCauseTransition( next_edge );
 
+    if( enable_will_toggle )
+    {
+        U64 enable_edge = mEnable->GetSampleOfNextEdge();
+        FrameV2 frame_v2_end_of_transaction;
+        mResults->AddFrameV2( frame_v2_end_of_transaction, "Disable", enable_edge, enable_edge + 1 );
```

Comment:
```suggestion
        mResults->AddFrameV2( frame_v2_end_of_transaction, "disable", enable_edge, enable_edge + 1 );
```

My only suggestion would be to use lowercase for the type to be consistent with the others.